index.html

```diff
@@ -0,0 +1,172 @@
+      </ul>
+      <h2>4</h2>
+    </div>
+
+    <ul class="b1">
+      <li>Bujar Berisha</li>
+      <li>Kim Gvozdic</li>
+      <li>Chris Hunter</li>
+      <li>Pascal Lampert</li>
+      <li>Alexandra Meyer &</li>
+      <li>Sevenn Perrig</li>
+    </ul>
+
+    <div class="b3">
+      <ul>
+        <li>Rochus Lussi</li>
+        <li>Heinz Stahlhut</li>
+      </ul>
+      <h2>5</h2>
+    </div>
+
+    <div class="c1">
+      <ul>
+        <li>Claudia Kubler</li>
+      </ul>
+      <h2 class="nine">9</h2>
+    </div>
+
+    <div class="c4">
+      <ul>
+        <li>Claudia Grimm &</li>
+        <li>darts</li>
+      </ul>
+      <h2 class="ten">10</h2>
+    </div>
+
+    <ul class="c6">
+      <li>Bebjarmin Heller</li>
+      <li>Anna-Sabina Zurrer</li>
+    </ul>
+
+    <div class="d3">
+      <ul>
+        <li>Leo Hofmann &</li>
+        <li>Benjamin van Bebber</li>
+      </ul>
+      <h2 class="eleven">11</h2>
+    </div>
+
+    <ul class="e2">
+      <li>Daniel Haller</li>
+      <li>Monica Germann</li>
+      <li>Maryina Lussi</li>
+    </ul>
+
+    <div class="e6">
+      <ul>
+        <li>Angela Anzi</li>
+        <li>Andrea Saemann</li>
+      </ul>
+      <h2 class="twelve">12</h2>
+    </div>
+
+    <div class="f1">
+      <ul>
+        <li>Bettina Diel</li>
+      </ul>
+      <h2 class="sixteen">16</h2>
+    </div>
+
+    <div class="f3">
+      <ul>
+        <li>Jens Nielsen</li>
+        <li>Judith Keller</li>
+      </ul>
+      <h2 class="seventeen">17</h2>
+    </div>
+
+    <ul class="f5">
+      <li>Rplf Hermann</li>
+    </ul>
+
+    <div class="g4">
+      <ul>
+        <li>alma</li>
+        <li>Lara Buffard</li>
+      </ul>
+      <h2 class="eighteen">18</h2>
+    </div>
+
+    <ul class="g6">
+      <li>Mio Chareteau</li>
+      <li>Martin Chramosta</li>
+      <li>nice</li>
+      <li>Sek 2a-Schule</li>
+      <li>Gersag2</li>
+      <li>Emmenbrucke</li>
+    </ul>
+
+    <div class="circle a1"></div>
+    <div class="circle a3"></div>
+    <div class="circle a5"></div>
+    <div class="circle b2"></div>
+    <div class="circle b4"></div>
+
+    <div class="circlesquare b6"></div>
+    <div class="circlesquare c2"></div>
+    <div class="circlesquare c5"></div>
+    <div class="circlesquare d1"></div>
+    <div class="circlesquare d4"></div>
+    <div class="circlesquare e1"></div>
+    <div class="circlesquare e3"></div>
+
+    <div class="square c3"></div>
+    <div class="square d2"></div>
+    <div class="square d5"></div>
+    <div class="square d6"></div>
+    <div class="square e4"></div>
+    <div class="square e5"></div>
+    <div class="square f2"></div>
+    <div class="square f4"></div>
+    <div class="square f6"></div>
+    <div class="square g1"></div>
+    <div class="square g3"></div>
+    <div class="square g5"></div>
+    <div class="square i6"></div>
+
+  </main>
+
+  <footer>
+    <h3 class="k1">akku Kunstplattform</h3>
+    <h3 class="k2">und</h3>
+  </footer>
+
+</body>
+
+</html>
```

Comment:
Your HTML is very well organized and clear. You've made a good use of semantic HTML!